apps/gro/fields/address-outside.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  'address-text-one': {
+    validate: ['required'],
+  }
+};
```

Comment:
No one if from outside the Uk , the other is from inside the Uk (it has the Town texbox, and a Postcode label and a link to allow you to change the Postcode.)